### ISSUE TITLE
AddToCart não adiciona promoção por UTM automaticamente

### ISSUE THREAD

**Issue by brenoc** [vtex/vtex.js#25]
**Você quer pedir uma nova feature ou reportar um bug?**
feature

**Qual o comportamento atual?**
AddToCart não adiciona promoção por UTM automaticamente

**Se o comportamento atual é um bug, por favor, forneça os passos para reproduzir o problema.**
Adicionar um cookie de UTM
Chamar a API de addToCart
A promoção do produto não será aplicada

**Qual o comportamento esperado?**
Chamar a API de addToCart e a promoção do produto ser aplicada automaticamente

**Qual browser e sistema operacional são afetados por esse problema?**
Todos

**Comment by brenoc** [comment]
O vtex.js faz isso hoje para manter consistencia com a API.

Para solucionar este problema, um `sendAttachment('marketingData', seuDadoAqui)` deve ser chamado posteriormente.

**Comment by rgoytacaz** [comment]
@brenoc 
bem discutível isso, o vtexjs é uma abstração pra facilitar o uso dessas apis.

O addToCart via GET, que é usado pelo portal (a url de adição ao carrinho) busca nos cookies as utms pra enviar o sendattachment. Hoje ela é meramente um proxy, e isso acaba sendo o comportamento esperado pelos clientes/parceiros usando o vtexjs.

Talvez vale a pena a gente abstrair também esse conceito para manter consistência com o portal?

**Comment by brenoc** [comment]
@rgoytacaz sim, concordo. Acho que vale a pena mudar o vtex.js para fazer isso.
